apps/rotm/index.js

```diff
@@ -47,7 +48,7 @@ module.exports = {
     },
     '/check-your-report': {
       prereqs: ['/image'],
-      behaviours: [require('hof-behaviour-summary-page'), 'complete', caseworkerEmailer],
+      behaviours: [require('hof-behaviour-summary-page'), 'complete', caseworkerEmailer, backLinkCheckReport],
```

Comment:
I think the behaviour should be called `checkReportBackLink`.

Reply:
do you mind if I ask why? I added the prefix backLink because that's the issue at hand here but I'm not fussed about it